server/service/src/programs/program_indicator/query.rs

```diff
@@ -153,32 +155,35 @@ impl ProgramIndicator {
 impl IndicatorLine {
     pub fn from_domain(line: IndicatorLineRow, columns: Vec<IndicatorColumnRow>) -> IndicatorLine {
         IndicatorLine {
-            name: line.description,
-            code: line.code,
+            name: line.description.clone(),
+            code: line.code.clone(),
             value: columns
                 .into_iter()
-                .map(|column| IndicatorColumn::from_domain(column))
+                .map(|column| IndicatorColumn::from_domain(column, line.id.clone()))
                 .collect(),
+            id: line.id,
         }
     }
 
     // TODO add to_domain utility function
 }
 
 impl IndicatorColumn {
-    pub fn from_domain(column: IndicatorColumnRow) -> IndicatorColumn {
+    pub fn from_domain(column: IndicatorColumnRow, line_id: String) -> IndicatorColumn {
```

Comment:
passing line_id down so that the loader in column can call with both line_id and column_id